Sidney-/slugtrade
# First start scripts

New Functionality:
- Added scripts for windows and mac/linux to start the server for the first time after cloning

How to test:
- Windows:
  - Blow away your repo and re-clone
  - Make sure you have python and pip set in your PATH
  - Run slugtrade/slug_trade/windows_first_start.bat
  - Ensure that the server starts and the site is populated with data

- Mac/Linux:
  - Blow away your repo and re-clone
  - Run slugtrade/slug_trade/first_start.sh
  - Ensure that the server starts and the site is populated with data
